pymc/distributions/continuous.py

```diff
@@ -3727,6 +3727,14 @@ def dist(cls, mu=0, sigma=1.0, *args, **kwargs):
 
         return super().dist([mu, sigma], *args, **kwargs)
 
+    def get_moment(rv, size, mu, sigma):
+        mu, sigma = at.broadcast_arrays(mu, sigma)
```

Comment:
```suggestion
```
This is not needed here. The parameters will be automatically broadcasted below as long as they show up in the same expression

Reply:
Removed. Thanks.